services/static-badge/static-badge.tester.js

```diff
@@ -62,3 +62,8 @@ t.create('Old static badge')
   .get('/foo/bar.png?color=blue', { followRedirect: false })
   .expectStatus(301)
   .expectHeader('Location', '/badge/foo-bar-blue.png')
+
+t.create('Old static badge without a color')
```

Comment:
Could you add a test of the behavior that you're fixing?

Reply:
I can. But as I wrote in PR description, this test is passing without a fix:
```js
t.create('Old static badge with a number as a color')
  .get('/foo/bar.png?color=1', { followRedirect: false })
  .expectStatus(301)
  .expectHeader('Location', '/badge/foo-bar-1.png')
```
A `color` value (in `queryParams`) is a string, not a number.

Reply:
In my opinion is it not worth to add a test which gives a false sense of working solution.

Reply:
Ahhh I understand now. For some reason when this request is made from the test, Scoutcamp is returning the query string param as a string, but when it's made to the real server it gets a number. That's really puzzling.

I'm hoping to replace Scoutcamp (#3328) so I don't feel like it's worth digging further into why it's happening.